uniresis/src/uniresis.cpp

```diff
@@ -89,14 +89,20 @@ class uniresis_t::updater_t : public std::enable_shared_from_this<uniresis_t::up
         subscope(),
         executor(),
         timer(executor.asio()),
-        log(std::move(log))
+        log(log)
     {}
 
+    ~updater_t() {
+        scope->close();
```

Comment:
Constructor doesn't initializes them, so SIGSEGVs nearby.

Reply:
yep, here you are right, will fix tomorrow.